### ISSUE TITLE
Three Card Poker Issues & Feature Request

### ISSUE THREAD

**Issue by Rxann** [YimMenu-Lua/Casino-Pacino#10]
I have noticed this issue for about 2 weeks, and the three card poker feature doesn't seem to actually give the deal bad beat or give me (and the rest of the players) royal flush. Also, would it be possible to get a feature for always getting blackjack? Sort of like the roulette feature but for blackjack.

**Comment by gir489returns** [comment]
You want it, you learn Lua and "fix" whatever issue you have.

**Comment by Rxann** [comment]
> You want it, you learn Lua and "fix" whatever issue you have.

I'll work on it eventually, I don't expect much success.

**Comment by Rxann** [comment]
Three Card Poker gives straight flush only on penthouse table, but it gives the dealer that too, so there is a bug in the iteration or total_players code

**Comment by gir489returns** [comment]
![image](https://github.com/YimMenu-Lua/Casino-Pacino/assets/100792176/4bcfc986-f9b2-426e-8341-773ed5e188da)

Nothing updated for the scripts with 3179, all the offsets that I can see are the same, so the script should work, something you're doing is incorrect.

**Comment by Rxann** [comment]
> ![image](https://github.com/YimMenu-Lua/Casino-Pacino/assets/100792176/4bcfc986-f9b2-426e-8341-773ed5e188da)
> 
> 
> 
> Nothing updated for the scripts with 3179, all the offsets that I can see are the same, so the script should work, something you're doing is incorrect.

Yeah I already tested for out of date globals and found nothing wrong like you did. All the code makes sense but something isn't working. I'll send a block of code that I think is the issue in a bit. I think it's related to the player count at the table though.

**Comment by Rxann** [comment]
I admit defeat, it works now.

**Comment by gir489returns** [comment]
"now"

![image](https://github.com/YimMenu-Lua/Casino-Pacino/assets/100792176/315d9abb-7edc-4a38-ad44-ede1effa8b50)
